### PR TITLE
Promoting SSH remotes and document their purpose, mentioning GitBox

### PR DESCRIPTION
**Pull request by attilapiros**

### CODE REVIEW

**Comment by srowen** [comment]
I think this is fine, but out of curiosity why are ssh: remotes better?

**Comment by attilapiros** [comment]
I run into an issue with https as they are deprecated. Got a mail from github when I tried to authenticate:

```
Hi @attilapiros,

You recently used a password to access the repository at apache/spark-website with git using git/2.21.1 (Apple Git-122.3).

Basic authentication using a password to Git is deprecated and will soon no longer work. Visit https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information around suggested workarounds and removal dates.

Thanks,
The GitHub Team
```

Visiting that link I found:
```
If you have two-factor authentication enabled for your account, you are already required to use token- or SSH-based authentication
```

**Comment by srowen** [comment]
OK I don't see that warning - I think I have password-less access setup for HTTPS repos (?) But it's fine to recommend in any event if there's no downside.

**Comment by attilapiros** [comment]
Thanks @srowen!

**Comment by attilapiros** [comment]
@srowen 

> I think this is fine,...

Does this mean you give a green light for this and I can merge this? We can wait for the others (or I can ask around the dev mailing list).

**Review by srowen — APPROVED** [review]
Yep go ahead and merge, fine by me

File: committers.md [hunk 1]
```diff
@@ -169,20 +169,21 @@ To use the `merge_spark_pr.py` script described below, you
 will need to add a git remote called `apache` at `https://github.com/apache/spark`,
```
Comment (zero323) [inline committers.md:169]: Should this be also adjusted?

File: committers.md [hunk 2]
```diff
@@ -169,20 +169,21 @@ To use the `merge_spark_pr.py` script described below, you
...
-origin	https://github.com/[your username]/spark.git (fetch)
-origin	https://github.com/[your username]/spark.git (push)
-upstream	https://github.com/apache/spark.git (fetch)
-upstream	https://github.com/apache/spark.git (push)
+apache	git@github.com:apache/spark-website.git (fetch)
+apache	git@github.com:apache/spark-website.git (push)
+apache-github	git@github.com:apache/spark-website.git (fetch)
+apache-github	git@github.com:apache/spark-website.git (push)
+origin	git@github.com:[your username]/spark-website.git (fetch)
+origin	git@github.com:[your username]/spark-website.git (push)
```
Comment (zero323) [inline committers.md:186]: Nitpick: we might want to keep `spark` not `spark-website`.

Comment (attilapiros) [reply committers.md:186]: I focused on the first commit which is writing your name on the website :).

Comment (attilapiros) [reply committers.md:186]: But you are right we should fix this especially the script is `merge_spark_pr.py`

**Comment by zero323** [comment]
>  run into an issue with https as they are deprecated. Got a mail from github when I tried to authenticate:

On a side note ‒ like I think it is not about https deprecation, but password authentication. I used https with username + oauth token on apache/spark without any issues or warnings.

**Comment by attilapiros** [comment]
> > run into an issue with https as they are deprecated. Got a mail from github when I tried to authenticate:
> 
> On a side note ‒ like I think it is not about https deprecation, but password authentication. I used https with username + oauth token on apache/spark without any issues or warnings.

Would you be so kind and update this page according your findings?

**Comment by zero323** [comment]
Not a problem, but let me do some more research first (I got here, trying to figure out if there is any hidden logic between assigning https to certain remotes and ssh to others). I used ssh to merge on spark-website, but haven't tried the same with main repo (just used user / token there).

**Pull request closed without merging**